docs/en/development/tests.md

```diff
@@ -39,14 +39,26 @@ The name of the test starts with a five-digit prefix followed by a descriptive n
 
 Some tests are marked with `zookeeper`, `shard` or `long` in their names. `zookeeper` is for tests that are using ZooKeeper. `shard` is for tests that requires server to listen `127.0.0.*`; `distributed` or `global` have the same meaning. `long` is for tests that run slightly longer that one second. You can disable these groups of tests using `--no-zookeeper`, `--no-shard` and `--no-long` options, respectively. Make sure to add a proper prefix to your test name if it needs ZooKeeper or distributed queries.
 
-### Checking for an Error that Must Occur
+### Checking for an error that must occur
```

Comment:
Most words are normally capitalized in titles.

https://grammar.yourdictionary.com/capitalization/rules-for-capitalization-in-titles.html